examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -100,12 +116,40 @@ class GalleryDrawer extends StatelessWidget {
             applicationLegalese: '© 2016 The Chromium Authors',
             aboutBoxChildren: <Widget>[
               new Padding(
-                padding: const EdgeInsets.only(top: 24.0, bottom: 8.0),
-                child: new Center(child: new CircularProgressIndicator())
-              ),
-              new Text('Awaiting fix for issue 4512.', textAlign: TextAlign.center),
+                padding: const EdgeInsets.only(top: 24.0),
+                child: new RichText(
+                  text: new TextSpan(
+                    children: <TextSpan>[
+                      new TextSpan(
+                        style: aboutTextStyle,
+                        text: "Flutter is an early-stage, open-source project to help "
+                        "developers build high-performance, high-fidelity, mobile "
+                        "apps for iOS and Android from a single codebase. This "
+                        "gallery is a preview of Flutter's many widgets, behaviors, "
+                        "animations, layouts, and more. Learn more about Flutter at "
+                      ),
+                      new LinkTextSpan(
+                        style: aboutLinkStyle,
+                        url: 'https://flutter.io'
+                      ),
+                      new TextSpan(
+                        style: aboutTextStyle,
+                        text: ".\n\nTo see the source code for this app, please visit "
+                      ),
+                      new LinkTextSpan(
+                        style: aboutLinkStyle,
+                        url: 'https://goo.gl/iv1p4G'
```

Comment:
Oh. I see. You're showing the URL itself in the dialog.

On Friday, June 24, 2016, Hans Muller <notifications@github.com> wrote:

> In examples/flutter_gallery/lib/gallery/drawer.dart
> <https://github.com/flutter/flutter/pull/4746#discussion_r68470524>:
>
> > +                        "developers build high-performance, high-fidelity, mobile "
> > +                        "apps for iOS and Android from a single codebase. This "
> > +                        "gallery is a preview of Flutter's many widgets, behaviors, "
> > +                        "animations, layouts, and more. Learn more about Flutter at "
> > +                      ),
> > +                      new LinkTextSpan(
> > +                        style: aboutLinkStyle,
> > +                        url: 'https://flutter.io'
> > +                      ),
> > +                      new TextSpan(
> > +                        style: aboutTextStyle,
> > +                        text: ".\n\nTo see the source code for this app, please visit "
> > +                      ),
> > +                      new LinkTextSpan(
> > +                        style: aboutLinkStyle,
> > +                        url: 'https://goo.gl/iv1p4G'
>
> I made it short because the long link looks terrible. A side benefit is
> that it gool.gl reports usage. If it's possible to change the link's
> value, it's not obvious (to me) from the goo.gl UI.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/flutter/flutter/pull/4746/files/3b77b85fe28882430d07f8f985e86a71a64ba13d#r68470524>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALTvi6il2KRa8mH1ebNWgiECXL4pckFmks5qPFaRgaJpZM4I-KbY>
> .
>